datanode/sqlstore/migrations/0058_transfers_fees_discount.sql

```diff
@@ -0,0 +1,23 @@
+-- +goose Up
+
+ALTER TABLE transfer_fees ADD COLUMN discount_applied HUGEINT;
```

Comment:
I would add an `IF NOT EXISTS` after the ADD COLUMN to be safe just in case a rollback fails for some reason.